_site/css/style.css

```diff
@@ -0,0 +1,369 @@
+html, body {
```

Comment:
@naveentherocker Looks like the indentation is not consistent throughout this file. Please check